### PR TITLE
Ll kth from end

### PR DESCRIPTION
**Pull request by mattburger**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/LinkedList.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package code401Challenges;
 
+import static java.lang.Math.abs;
+
 public class LinkedList {
     Node head = null;
 //    Node tail;
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/LinkedList.java]: Corpse code!

File: code401Challenges/src/main/java/code401Challenges/LinkedList.java [hunk 2]
```diff
@@ -87,6 +89,39 @@ protected void insertAfter(Object value, Object newValue){
...
+            int count = 0;
+            int length = 0;
+            k = abs(k); //absolute value of k
+
+            while(ref != null){
+                if(count == k){
+                    ref = ref.next;
+                    main = main.next;
+                    length++;
+                }
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/LinkedList.java]: `} else {`
